tensorflow/python/kernel_tests/check_ops_test.py

```diff
@@ -1557,6 +1557,13 @@ def test_sparsetensor_raises_when_wrong_type(self):
     with self.assertRaisesRegexp(TypeError, "must be of type.*float32"):
       check_ops.assert_type(sparse_float16, dtypes.float32)
 
+  def test_raise_when_tf_type_is_not_dtype(self):
+    # Test case for GitHub issue 45975
```

Comment:
Let's replace the number with the URL so it's easy to get to the issue.

Reply:
@mihaimaruseac The PR has been updated.